File: Basics/While-Loops/Exercise/Resources/the_old_library.py
searched_book = input()
checked_books_count = 0
is_book_found = False
checked_book = input()

while checked_book != "No More Books":
    if checked_book == searched_book:
        is_book_found = True
        break

    checked_books_count += 1
    checked_book = input()

if is_book_found:
    print(f"You checked {checked_books_count} books and found it.")
else:
    print(f"The book you search is not here!\n"
          f"You checked {checked_books_count} books.")

File: JavaScript/Basics/0.5-While-Loops/Exercises/the-old-library.js
function findBook(input) {
    let searchedBook = input.shift(); let i = 0;
    while (i < input.length) {
        if (input[i] == searchedBook) {return `You checked ${i} books and found it.`}
        if (input[i] == "No More Books") {break;}
        i++
    }
    return `The book you search is not here!\nYou checked ${i} books.`;
}


console.log(findBook(["Troy",
"Stronger",
"Life Style",
"Troy"]));console.log(findBook(["Bourne",
"True Story",
"Forever",
"More Space",
"The Girl",
"Spaceship",
"Strongest",
"Profit",
"Tripple",
"Stella",
"The Matrix",
"Bourne"]));
console.log(findBook(["The Spot",
"Hunger Games",
"Harry Potter",
"Torronto",
"Spotify",
"No More Books"]));
console.log(findBook(["Bourne",
"True Story",
"Forever",
"More Space",
"The Girl",
"Spaceship",
"Strongest",
"Profit",
"Tripple",
"Stella",
"The Matrix",
"Bourne"]));